### DISCUSSION TITLE
Regarding contributing

### DISCUSSION THREAD

**Discussion by Kushagrasaxena-13** [audacity/audacity#1666] [General]
Hey contributors , I want to contribute to this project but I am new to this can anyone guide me.

**Comment by LWinterberg** [comment]
There's several ways to contribute, I just added some options you can do here: https://github.com/audacity/audacity/blob/contributeinfo/CONTRIBUTING.md 

What do you want to do exactly?

**Reply by Kushagrasaxena-13** [reply]
I want to tackle good first issue as I am new to this

**Comment by Kushagrasaxena-13** [comment]
hey contributors , I had tried to install it but after configuring in CMAKE its showing error "**error in configuring process ,project files may be invalid** ". I am a windows user. please help me out

**Comment by JohnColket** [comment]
@Kushagrasaxena-13, join Audacity-Dev on discord here: https://discord.gg/KT7Px4jc (expiring link) and find some help on #windows.
